sdk/src/main/scala/io/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -1035,4 +1037,16 @@ class EthService(
       }
     }
   }
+
+  @RpcMethod("web3_sha3")
+  def getSHA3(data: String): String = {
```

Comment:
```suggestion
  def getSHA3(data: Array[Byte]): String = {
```
we may expect byte array type here, json unmarshaller will automatiically apply `EthByteDeserializer` to this type and process `0x` properly. Note: it allows hex string with or without `0x`.

Reply:
I haven't found an explicit requirement about the 0x prefix in input param, but trying web3_sha3 on an ethereum node with an hex string without 0x returns an error ("cannot unmarshal hex string without 0x prefix into Go value of type hexutil.Bytes"). 
I don't know if we should be as "strict" or not, to be honest.